.gitignore

```diff
@@ -98,3 +98,9 @@ bagitobjecttransfer/static/
 .dockerenv
 *.sqlite3
 env/
+package-lock.json
+node_modules
+
+# Temporary ARM docker compose files to be removed later
+compose-arm.dev.yml
+compose-arm.prod.yml
```

Comment:
Can you remove these lines, and not stage those compose files rather than add them to the .gitignore?

Reply:
Addressed by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/186/commits/946c4cf235119f99741cb85f6eacb4fb058cd6cd